src/utils/LogicUtils.res

```diff
@@ -492,6 +492,8 @@ let isEmptyDict = dict => {
 
 let isEmptyString = str => str->String.length === 0
 
+let isNonEmptyString = str => str->String.length > 0
```

Comment:
This is not req operations can be performed via doing ~not of the above. Just a recommendation

Reply:
its better to have for better readability